src/utils.ts

```diff
@@ -23,9 +23,9 @@ function formatDate(date: Date) {
   return `${date.getFullYear()}-${month}-${day} ${hours}:${minutes}:${seconds}.${ms}`;
 }
 
-export function logPrefix(logLevel: LogLevel) {
+export function logPrefix(_logLevel?: LogLevel) {
```

Comment:
remove _logLevel

Reply:
This is a lot changes in the code, I'll keep it inactive in case we'll decide to bring back the type labels